CMakeLists.txt

```diff
@@ -39,6 +39,15 @@ else()
   set (EXTRA_TEST_LIB_DEPS)
 endif()
 
+set(skip_pybind11_default_value OFF)
+if (MSVC)
+  set(skip_pybind11_default_value ON)
+endif()
+
+option(SKIP_PYBIND11
```

Comment:
I think we'll need to use this variable as a condition where the pybind11 target is created. See https://github.com/gazebosim/gz-math/pull/528